### PR TITLE
Review styling

### PR DESCRIPTION
**Pull request by aesuan**

### CODE REVIEW

File: client/src/components/sections/Review.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  margin: 1em;
+  padding: 2em;
+  border: 1px solid grey;
+`;
+
+// function Meta (props) {
+//   return (
+//     <>
+//       <Helpful />
+//       <Button onClick={handleHelpful}>"Helpful"</Button>
```
Comment (erichen886) [inline client/src/components/sections/Review.jsx:45]: I might be missing something but what is {handleHelpful}

File: client/src/utils/api.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// @flow
+import axios from 'axios';
+
+const endpoint: string = '/api';
+
+export function getRandomProduct(): Promise<Object> {
```
Comment (erichen886) [inline client/src/utils/api.jsx:6]: I'm intrigued by this syntax. I haven't seen it before.

File: client/src/components/sections/Review.jsx (hunk 1 above)
Comment (aesuan) [reply client/src/components/sections/Review.jsx:45]: this is all commented out it's not part of the code yet!

File: client/src/utils/api.jsx (hunk 2 above)
Comment (aesuan) [reply client/src/utils/api.jsx:6]: it's typechecking with flow!

**Pull request merged**